sdk/storage/azure-storage-blobs/test/ut/blob_sas_test.cpp

```diff
@@ -530,7 +530,7 @@ namespace Azure { namespace Storage { namespace Test {
     VerifyBlobSasNonRead(blobClient, sasToken);
   }
 
-  TEST_F(BlobSasTest, AccountSasService)
+  TEST_F(BlobSasTest, AccountSasService_LIVEONLY_) // keeps failing in CI
```

Comment:
I think you'll have to make a new recording.

If you cannot fix this anyway, can you open a GitHub issue? So that storage team won't lose track of this.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/5229